src/tests/unit/tests/DetailsView/components/issue-filing-dialog.test.tsx

```diff
@@ -91,12 +86,15 @@ describe('IssueFilingDialog', () => {
     });
 
     it('render: validate correct callbacks (file issue on click and cancel/dismiss)', () => {
-        isSettingsValidMock.setup(isSettingsValid => isSettingsValid(selectedServiceData)).verifiable();
+        isSettingsValidMock
+            .setup(isSettingsValid => isSettingsValid(selectedServiceData))
+            .returns(() => true)
+            .verifiable();
         createBugFilingUrlMock
             .setup(createBugFilingUrl => createBugFilingUrl(selectedServiceData, selectedBugDataStub, envInfo))
             .verifiable();
         telemetryCallbackMock.setup(telemetryCallback => telemetryCallback(eventStub)).verifiable();
-        onCloseMock.setup(onClose => onClose(null)).verifiable();
+        onCloseMock.setup(onClose => onClose(It.isAny())).verifiable(Times.exactly(2));
```

Comment:
why use `It.isAny()`? I think we are explicitly passing `null` on line 101, right?